Content.Client/Toilet/ToiletSystem.cs

```diff
@@ -0,0 +1,31 @@
+using Content.Shared.Cabinet;
+using Content.Shared.Toilet;
+using Content.Shared.Verbs;
+using Robust.Client.GameObjects;
+
+namespace Content.Client.Toilet;
+
+public sealed class ToiletSystem : SharedToiletSystem
+{
+    protected override void UpdateAppearance(EntityUid uid, ToiletComponent? component = null)
+    {
+        if (!Resolve(uid, ref component))
+            return;
+
+        if (!TryComp<SpriteComponent>(uid, out var sprite))
+            return;
+
+        var state = component.ToggleSeat ? component.SeatUp : component.SeatDown;
+        var lidstate = component.ToggleLid ? component.LidOpen : component.LidClosed;
+        if (state != null)
+            sprite.LayerSetState(ToiletVisualLayers.Door, state);
+        if (lidstate != null)
+            sprite.LayerSetState(ToiletVisualLayers.Lid, lidstate);
+    }
```

Comment:
why is this an override of a virtual function instead of just being on the client only? server never overrides it

Reply:
Im a little ocnfused on how to reference the appearancechange if i cannot override the shared function with the client one.

Reply:
should probably use generic visualizer with seat enum i think it would work

Reply:
> should probably use generic visualizer with seat enum i think it would work

tried it but just could not get it to work.

Reply:
Used a generic Visualizer and deleted the client visualizer system.